README.rst

```diff
@@ -74,6 +76,18 @@ For migration purposes, it is possible to transfer all public playlists of a use
 
     spotify_to_ytmusic all <spotifyuserid>
 
+Transfer liked tracks of the Spotify user
+-----------------------------------------
+
+**You must you oAuth authentification for transfering liked songs.**
+
+.. code-block::
+
+   spotify_to_ytmusic liked
+
+This command will open browser where you should give access to your account (if you havn't done that before). 
+After authorization you will be redirected to localhost, copy link you were redirected to (looks like localhost/?code=...) and paste to command line.
```

Comment:
I think this requires setting the correct redirect uri in the Spotify dashboard? Please state such here.

Reply:
I'm not sure I haven't set any redirect uri and successfully transfer my tracks. Probably Spotify add localhost by default. I will update readme and mention that.